<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export 3D File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="6" column="0">
    <widget class="QLabel" name="label_method">
     <property name="text">
      <string>Export method:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QVBoxLayout" name="layer_layout"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Layer Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QCheckBox" name="bool_modelviewer">
     <property name="text">
      <string>Export model-viewer HTML</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="combosel_compression"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="listsel_layer"/>
   </item>
   <item row="19" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ok">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_filetype_message">
     <property name="text">
      <string>Filetype</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="combosel_filetype"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;File Options&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_compression_message">
     <property name="text">
      <string>Compression method</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_settings_message">
     <property name="text">
      <string>Select export options</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="combosel_method"/>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QCheckBox" name="bool_layer_controls">
     <property name="text">
      <string>Include model-viewer layer controls</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
